engine/graphics/src/graphics.h

```diff
@@ -216,33 +219,34 @@ namespace dmGraphics
         , m_VWrap(TEXTURE_WRAP_CLAMP_TO_EDGE)
         , m_Data(0x0)
         , m_DataSize(0)
-        , m_MipMap(0)
+        , m_X(0)
+        , m_Y(0)
+        , m_Z(0)
         , m_Width(0)
         , m_Height(0)
         , m_Depth(0)
+        , m_MipMap(0)
         , m_SubUpdate(false)
-        , m_X(0)
-        , m_Y(0)
-        , m_Z(0)
         {}
 
         TextureFormat m_Format;
         TextureFilter m_MinFilter;
         TextureFilter m_MagFilter;
-        TextureWrap m_UWrap;
-        TextureWrap m_VWrap;
-        const void* m_Data;
-        uint32_t m_DataSize;
-        uint16_t m_MipMap;
-        uint16_t m_Width;
-        uint16_t m_Height;
-        uint16_t m_Depth; // For array texture, this is slice count
+        TextureWrap   m_UWrap;
+        TextureWrap   m_VWrap;
+        const void*   m_Data;
+        uint32_t      m_DataSize;
```

Comment:
Sorted on size

Reply:
Not all the way though.
The pointer is potentially larger than the enums.

Reply:
Fixed